### PR TITLE
DRAFT: building content

### PR DESCRIPTION
**Pull request by ElenaFdR**
To do:
- [x] add README.md
- [x] add introduction-en.md
- [x] add introduction-nl.md
- [x] add questions-en.md
- [x] add questions-nl.md
- [x] how-to-respond-en.md
- [x] how-to-respond-nl.md
- [x] change links from HackMD to Github
- [ ] draft governance document

### CODE REVIEW

**Review by Ainali — APPROVED** [review]

File: how-to-respond-en.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+| <a href="README.md">Home</a> | <a href="introduction-en.md">Introduction</a> | <a href="introduction-en.md">Our proposals and questions</a> | How to respond |
```
Comment (clausmullie) [inline how-to-respond-en.md]: <a href="introduction-en.md">Our proposals and questions</a>

Link to be fixed

File: how-to-respond-en.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+| Market consultation publication | 14-4-2020 |
+| Submission deadline | 1-5-2020, 17:00 |
+| Possible invitation to workshop | 13-05-2020 |
+| Workshop (invitation only) | 20/21-5-2020 |
+
+## Legal aspects
+
+This document is for market consultation purposes only. It is made available on the express condition that it will only be used to provide suggestions and ideas. Participation in this market consultation cannot lead to obligations on the part of the participating market parties to enter into agreements with the participating municipalities or Dimpact, and vice versa. Market consultation participation does not entail any reimbursement of costs.
+
+| <a href="README.md">Home</a> | <a href="introduction-en.md">Introduction</a> | <a href="introduction-en.md">Our proposals and questions</a> | How to respond |
```
Comment (clausmullie) [inline how-to-respond-en.md]: <a href="introduction-en.md">Our proposals and questions</a>

Link to be fixed

File: introduction-nl.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Marktpartijen die geïnteresseerd zijn in deelname worden gevraagd [onze voorstellen en vragen](https://hackmd.io/OFGiBKmLSh6ug7Gpv3Hbzw?both) zorgvuldig te herzien en te reageren op de daar genoemde vragen.
+
+De antwoorden moeten voor 1-5-2020 worden ontvangen door Niels Lindeboom. Op basis van deze antwoorden kunnen marktpartijen samen met de gemeenten worden uitgenodigd voor één of meerdere workshops om te bepalen hoe de OpenZaak-gemeenschap kan worden opgezet. Meer informatie over dit marktraadplegingsproces is te vinden in [hoe te reageren](https://hackmd.io/NT4ywg6bSReSgsocu1HFXw?both).
+
+Deze marktconsultatie wordt georganiseerd door [Dimpact](https://www.dimpact.nl/), in opdracht van de OpenZaak gemeenten, ondersteund door de [Foundation for Public Code](https://publiccode.net/).
+
+<div class="navoz"><a href="README.md">Home</a> | Inleiding | <a href="questions-nl.md">Onze voorstellen en vragen</a> | <a href="how-to-respond-nl.md">Reageren op de marktconsultatie</a></div>
+
+<style>
```
Comment (clausmullie) [inline introduction-nl.md]: Not rendering well?

**Review by clausmullie — CHANGES_REQUESTED** [review]
Mainly checked the hyperlinks

File: introduction-en.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 # Introduction: Open invitation for market parties to join the OpenZaak community
 
-<div class="navoz"><a href="https://hackmd.io/b6mszG6HRjyDGRCi72_y2w">Home</a> | Introduction | <a href="https://hackmd.io/OFGiBKmLSh6ug7Gpv3Hbzw">Our proposals and questions</a> | <a href="https://hackmd.io/AQOmgrpSQIeMVC4OhFxKDA">How to respond</a></div>
+| <a href="https://hackmd.io/b6mszG6HRjyDGRCi72_y2w">Home</a> | Introduction | <a href="https://hackmd.io/OFGiBKmLSh6ug7Gpv3Hbzw">Our proposals and questions</a> | <a href="https://hackmd.io/AQOmgrpSQIeMVC4OhFxKDA">How to respond</a> |
```
Comment (clausmullie) [inline introduction-en.md]: | <a href="https://hackmd.io/b6mszG6HRjyDGRCi72_y2w">Home</a> | Introduction | <a href="https://hackmd.io/OFGiBKmLSh6ug7Gpv3Hbzw">Our proposals and questions</a> | <a href="https://hackmd.io/AQOmgrpSQIeMVC4OhFxKDA">How to respond</a> |

Still links to hackmd?

File: introduction-en.md [hunk 5]
```diff
@@ -26,21 +27,5 @@ Answers must be received by Niels Lindeboom before 1-5-2020. Based on these answ
...
-border: 1px solid #bce8f1; 
-background: #D8ECF7; 
-text-align: left; 
-font-size: 100%; 
-line-height: 1.5em; 
-border-radius: 5px; 
-color: #31708f;
-}
-</style>
+| <a href="https://hackmd.io/b6mszG6HRjyDGRCi72_y2w">Home</a> | Introduction | <a href="https://hackmd.io/OFGiBKmLSh6ug7Gpv3Hbzw">Our proposals and questions</a> | <a href="https://hackmd.io/AQOmgrpSQIeMVC4OhFxKDA">How to respond</a> |
```
Comment (clausmullie) [inline introduction-en.md]: | <a href="https://hackmd.io/b6mszG6HRjyDGRCi72_y2w">Home</a> | Introduction | <a href="https://hackmd.io/OFGiBKmLSh6ug7Gpv3Hbzw">Our proposals and questions</a> | <a href="https://hackmd.io/AQOmgrpSQIeMVC4OhFxKDA">How to respond</a> |

Still links to hackmd?

**Review by clausmullie — CHANGES_REQUESTED** [review]
Think some more links need to be updated

File: introduction-nl.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+## Een gezonde leverancierscommunity
+
+De deelnemende gemeenten zijn op zoek naar geïnteresseerde leveranciers om hen te ondersteunen bij de implementatie van de codebase (onder meer in de ontwikkeling, onderhoud, implementatie en ondersteuning). De ambitie is om een gezonde open source community te hebben waarin leveranciers de mogelijkheid krijgen om de markt voor OpenZaak te laten groeien, en waarin inkoop wordt vereenvoudigd en gestandaardiseerd.
+
+Hiervoor vinden we het belangrijk om een open en collaboratieve gemeenschap te hebben, inclusief één codebase met een gemeenschappelijke governancestructuur en een gedeeld begrip over bugfixes en -ondersteuning. Meer informatie over de behoeften, conceptvisie en open vragen van de gemeenten is te vinden in [onze voorstellen en vragen](https://hackmd.io/OFGiBKmLSh6ug7Gpv3Hbzw) en het [draft governance.md document](governance.md).
```
Comment (clausmullie) [inline introduction-nl.md]: hackmd link

File: introduction-nl.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+## Over deze marktconsultatie
+
+Marktpartijen die geïnteresseerd zijn in deelname worden gevraagd [onze voorstellen en vragen](https://hackmd.io/OFGiBKmLSh6ug7Gpv3Hbzw?both) zorgvuldig te herzien en te reageren op de daar genoemde vragen.
```
Comment (clausmullie) [inline introduction-nl.md]: hackmd link

File: introduction-nl.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+De antwoorden moeten voor 1-5-2020 worden ontvangen door Niels Lindeboom. Op basis van deze antwoorden kunnen marktpartijen samen met de gemeenten worden uitgenodigd voor één of meerdere workshops om te bepalen hoe de OpenZaak-gemeenschap kan worden opgezet. Meer informatie over dit marktraadplegingsproces is te vinden in [hoe te reageren](https://hackmd.io/NT4ywg6bSReSgsocu1HFXw?both).
```
Comment (clausmullie) [inline introduction-nl.md]: hackmd link

**Review by clausmullie — CHANGES_REQUESTED** [review]
Few more links to rewire

**Review by clausmullie — APPROVED** [review]

**Review by AlbaRoza — COMMENTED** [review]
Looks good! :)

**Pull request merged**